Makefile

```diff
@@ -89,6 +90,10 @@ endif
 build: fmt vet
```

Comment:
Shouldn't doing `make build` also result in building WICD? Otherwise you will need to add another explicit entry to build this in the release repo. Adding `build-wicd` will achieve that here which will also give you a CI signal as to whether it compiles or not.

Reply:
Good point. I will build the daemon under `build` instead of creating a new target.

Reply:
Added

Reply:
> Good point. I will build the daemon under `make build` instead of creating a new target.

It is fine to have a separate target. That has the added flexibility of being able to compile just the daemon. In fact we might want to add a separate target to build the operator too.

Reply:
+1 to separate make target

Reply:
I dont see a reason to have an entry in the release repo for this

Reply:
why do we need to have an explicit entry in the release repo? Lets keep release repo change as minimal as possible.

Reply:
I don't think it is needed either
![Screenshot from 2022-03-08 10-32-24](https://user-images.githubusercontent.com/66272285/157272452-4a7694ca-de74-41e5-a5d0-21d1dc874f2b.png)
. When running `make build-daemon` the binary is saved to the _output/bin directory.